### DISCUSSION TITLE
Renovate upgrading ember-cli past the current LTS despite using ~3.20.2

### DISCUSSION THREAD

**Discussion by serenecloud** [renovatebot/renovate#8272] [Request Help] [answered]
**What Renovate type, platform and version are you using?**

Self hosting renovate:23.66 in gitlab CE.

**Describe the issue**

When adding Renovate to our Ember.js (and python) project our package.json devDependencies version for `ember-cli` was `~3.20.2`. Based on my understanding this should limit upgrades to 3.20.x (the current LTS) but no higher.

The "Configure Renovate" MR told us it would upgrade `ember-cli` to `3.24.0` and after we merged the "pin dependencies" MR it made a MR for that upgrade. We have closed that MR without merging.

It's possible this can be changed with config, but it appears that Renovate is upgrading past our desired LTS release version and getting the latest 3.x version.

I wanted to see if this was a configuration issue on our end before opening a bug. We have allowed renovate to pin our package.json file now, so it is set to 3.20.2 - what would be the best way to get Renovate to generate a `3.20.6` merge request?

Our renovate.json:
```
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "pip_requirements": {
    "fileMatch": [
      "app/requirements/base.txt",
      "app/requirements/dev.txt"
    ]
  },
  "prHourlyLimit": 5,
  "extends": [
    "docker:disable"
  ]
}
```
config.js:

```
module.exports = {
  platform: 'gitlab',
  endpoint: 'https://our-gitlab/api/v4/',
  assignees: ['serenecloud'],
  baseBranches: ['master'],
  labels: ['renovate'],
  rebaseWhen: 'auto',
  automerge: false,
  automergeType: 'branch',
  semanticCommits: "enabled",
  major: {
    automerge: false
  },
  repositories: [
    "client/repo",
    "client/other-repo"
  ],
  extends: [
    'config:base',
    ":pinAllExceptPeerDependencies"
  ]
};
```

Thanks very much.

**Comment by viceice** [comment, marked as answer]
You need [`separateMinorPatch=true`](https://docs.renovatebot.com/configuration-options/#separateminorpatch)

btw: update renovate, latest is 24.16.2 as of now, see https://hub.docker.com/r/renovate/renovate/tags?page=1&ordering=last_updated

**Comment by rarkins** [comment]
Where/how is that "LTS" information defined? Do I understand it correctly that although `3.24.0` is a *stable* version according to SemVer/npm, this version of ember-cli is considered *unstable*?

Also a simple reproduction repo would be required to debug further.

**Comment by serenecloud** [comment]
@viceice that's the setting we were after, thanks so much, I'm getting a lot more minor point release MRs now.

@rarkins The web page https://emberjs.com/releases/lts is where I get the assertion that 3.20 is the current LTS - https://www.npmjs.com/package/ember-cli also has an lts tag.

As best I can tell from release notes https://blog.emberjs.com/ember-3-20-released/ and https://blog.emberjs.com/ember-3-24-released/ both 3.20 and 3.24 are LTS candidates, but currently 3.20 is the current LTS. I wasn't aware of this before yesterday but our Ember expert flagged it so I started looking at it.
